src/general/OptionsMenu.tscn

```diff
@@ -635,30 +637,61 @@ __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="HSeparator" type="HSeparator" parent="CenterContainer/VBoxContainer/HBoxContainer"]
+[node name="Reset" type="Button" parent="CenterContainer/VBoxContainer/HBoxContainer"]
 margin_left = 104.0
-margin_right = 241.0
+margin_right = 181.0
+margin_bottom = 35.0
+disabled = true
+text = "Reset"
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="Save" type="Button" parent="CenterContainer/VBoxContainer/HBoxContainer"]
+margin_left = 185.0
+margin_right = 285.0
+margin_bottom = 35.0
+rect_min_size = Vector2( 100, 35 )
+disabled = true
+text = "Save"
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="HSeparator" type="HSeparator" parent="CenterContainer/VBoxContainer/HBoxContainer"]
+margin_left = 289.0
+margin_right = 442.0
 margin_bottom = 35.0
 size_flags_horizontal = 3
 custom_styles/separator = SubResource( 6 )
 
-[node name="Reset" type="Button" parent="CenterContainer/VBoxContainer/HBoxContainer"]
-margin_left = 245.0
+[node name="Defaults" type="Button" parent="CenterContainer/VBoxContainer/HBoxContainer"]
+margin_left = 446.0
 margin_right = 549.0
 margin_bottom = 35.0
-text = "Reset ALL to Default Settings"
+text = "Defaults"
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="ResetConfirm" type="ConfirmationDialog" parent="CenterContainer"]
+[node name="BackConfirm" type="ConfirmationDialog" parent="CenterContainer"]
+margin_left = 323.0
+margin_top = 324.0
+margin_right = 957.0
+margin_bottom = 395.0
+rect_min_size = Vector2( 600, 70 )
+popup_exclusive = true
+window_title = ""
```

Comment:
Is this popup missing a title?

Reply:
The defaults popup had a title that was more like a redundant question so I thought it was simpler to just have no title on either of them, but I can add them if you like.

Reply:
I would like to have a really short question title. With the actual window text content providing more information.